gradio/interface.py

```diff
@@ -144,6 +145,7 @@ def __init__(
         | None = None,
         time_limit: int | None = 30,
         stream_every: float = 0.5,
+        deep_link: str | DeepLinkButton | None = None,
```

Comment:
We should add docstring for this param

Reply:
Also, as it's currently implemented, there's no way for users to disable the deep link for their Gradio app on Spaces. I think we should allow this, for example if a user sets `deep_link=False`

Reply:
Also, if I want to "just enable it" in my Interface running locally, I should be able to set `deep_link=True` without having to provide a `str`